packages/nouns-contracts/script/executorV3AndExcessETHBurner/ProposeExecutorV3UpgradeBase.s.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.15;
+
+import 'forge-std/Script.sol';
+
+interface INounsDAOLogicV3 {
+    function propose(
+        address[] memory targets,
+        uint256[] memory values,
+        string[] memory signatures,
+        bytes[] memory calldatas,
+        string memory description
+    ) external returns (uint256);
+}
+
+abstract contract ProposeExecutorV3UpgradeBase is Script {
```

Comment:
@davidbrai do you want to update this script to do more stuff, to basically align it with your mainnet fork test?

would be good to use as many scripts as possible as part of that flow

Reply:
@eladmallel see my reply below regarding not doing the proposal part via script